packages/purpose-process/src/services/purposeService.ts

```diff
@@ -966,15 +992,15 @@ const replacePurposeVersion = (
   };
 };
 
-const getInvolvedTenantByEServiceMode = async (
+const getInvolvedTenantKindByEServiceMode = async (
```

Comment:
What about `retrieveKindOfInvolvedTenantByEServiceMode`? (More verbose but cleaner in my opinion)